### PR TITLE
Potato

### PR DESCRIPTION
**Pull request by MillerNerd**
Mmm nummy

### CODE REVIEW

**Comment by smiller171** [comment]
stop killing my readme @MillerNerd

**Comment by MillerNerd** [comment]
I don't give a fuck :D:D:D

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +0,0 @@
-FROM alpine
-
-MAINTAINER Scott Miller <scott.miller171@gmail.com>
```
Comment (smiller171) [inline Dockerfile:3]: Why Do?

File: README.md [hunk 2]
```diff
@@ -1,9 +1 @@
-# alpine-python
-
-[![Build Status](https://travis-ci.org/smiller171/alpine-python.svg)](https://travis-ci.org/smiller171/alpine-python)
-
-[![Join the chat at https://gitter.im/smiller171/alpine-python](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/smiller171/alpine-python?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
-
-<a href="https://zenhub.io"><img src="https://raw.githubusercontent.com/ZenHubIO/support/master/zenhub-badge.png"></a>
-
-Docker image with python, python-pip, python-dev, and gcc on alpine linux
+I DON'T KNOW HOW TO POTATO!!!
```
Comment (smiller171) [inline README.md:1]: Bullshit

File: Dockerfile (hunk 1 above)
Comment (MillerNerd) [inline Dockerfile:3]: Boop

**Comment by smiller171** [comment]
For anyone who sees this later, I was teaching my brother how to PR

**Comment by MillerNerd** [comment]
And how to git in general because I'm totally not a git n00b at all...
Yeah, that's right, be jelly :P

**Pull request closed without merging**